src/main/mw.c

```diff
@@ -338,6 +338,11 @@ void mwArm(void)
     }
 }
 
+int32_t getRcStickPosition(int32_t axis) {
+
+    return min(abs(rcData[axis] - masterConfig.rxConfig.midrc), 500);
```

Comment:
IMO abs() is bad idea here - name it `getRcStickDeflection` or return negative value ...